### PR TITLE
general changes to docs and README.md

### PR DESCRIPTION
**Pull request by ItsbaileyX3525**

### CODE REVIEW

File: docs/documentation.html [hunk 1]
```diff
@@ -91,14 +91,15 @@
 
 .tileset_size
 .tile_coordinate
-
+<!--Cannot be invoked/inherited.-->
```
Comment (pokepetter) [inline docs/documentation.html]: ?

File: docs/documentation.html [hunk 2]
```diff
@@ -76,11 +76,11 @@
 .text_color
 .text_size
 .text_origin
-.fit_to_text()
+.fit_to_text() <gray># Creates background and fits text to it</gray>
 
-.on_click
-.on_enable
-.on_disable
+.on_click <gray># To check if button clicked</gray>
```
Comment (pokepetter) [inline docs/documentation.html]: This is not to check if a button has been clicked. It's to bind a function to be called when you click the button.

File: README.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ Sunsnake is a Python inspired language for the web.
 * Indentation significant, no more {}-brackets :)
 * Significantly faster than Python.
 * Supports macros.
+* Supports controller input.
```
Comment (pokepetter) [inline README.md]: This is not a language feature. It's a separate "module" you can use with Taptapir

File: docs/documentation.html (hunk 1 above)
Comment (ItsbaileyX3525) [reply docs/documentation.html]: Originally from it being a class you'd think you can inherit the class like you can in ursina which was dead confusing to being with

File: docs/documentation.html (hunk 2 above)
Comment (ItsbaileyX3525) [reply docs/documentation.html]: ah right my apologies, i don't know how i messed that up

File: README.md (hunk 3 above)
Comment (ItsbaileyX3525) [reply README.md]: oh, i just assumed you wrote it too

**Pull request closed without merging**